generator/src/test/java/com/scottlogic/deg/generator/cucumber/OfLength.feature

```diff
@@ -516,7 +511,6 @@ Feature: User can specify the length of generated string data using 'ofLength'
       | foo  |
       | "22" |
 
-   @ignore # issue 246
```

Comment:
This is a string constraint combined with aValid, does this throw an error. Is there benefit in the test?

Reply:
This test does not fail, which maybe doesn't fit in with issue #487 as would be expected. I also think this test is covered in the Valid.feature by the following scenario: 
`Scenario Outline: Running an 'aValid' request alongside a contradicting ofLength constraint should only emit null
    Given foo is a valid "ISIN"
    And foo is of length <length>
    Then the following data should be generated:
      | foo  |
      | null |
    Examples:
    | length |
    | 11     |
    | 13     |
    | 1      |
    | 0      |
    | 9999   |`

I am going to delete this scenario given the reason above.